--- ./src/Base.c.orig	Tue Jul 23 12:57:24 2002
+++ ./src/Base.c	Tue Jul 23 21:48:28 2002
@@ -47,7 +47,7 @@
    NBugReset();
 
    app_path=QDir::home().path();
-   image_path=QString("/opt/MathPlanner3/pixmaps");
+   image_path=QString("/usr/X11R6/share/MathPlanner3/pixmaps");
 
    QTime time=QTime::currentTime();
    sprintf(session,"%d",time.second());
--- ./src/ConfigReader.cpp.orig	Tue Jul 23 12:57:24 2002
+++ ./src/ConfigReader.cpp	Tue Jul 23 21:49:08 2002
@@ -173,7 +173,7 @@
   return;
 	}
 
-  file.setName("/opt/MathPlanner3/MPLConfig.txt");
+  file.setName("/usr/X11R6/share/MathPlanner3/MPLConfig.txt");
 
   if (file.exists()) {
 	file.open( IO_ReadOnly );
--- ./src/Math.cpp.orig	Tue Jul 23 12:57:24 2002
+++ ./src/Math.cpp	Tue Jul 23 21:50:29 2002
@@ -83,7 +83,7 @@
 {
 	ToolBarB=new QToolBar(this);
   /*
-	KIconLoader kicon(QString("/opt/MathPlanner"));
+	KIconLoader kicon(QString("/usr/X11R6/share/MathPlanner3"));
 
   pm1=kicon.loadIcon("filenew",KIcon::Toolbar);
   pm2=kicon.loadIcon("fileopen",KIcon::Toolbar);
