--- ./src/Makefile.org	Tue Jul 23 12:57:24 2002
+++ ./src/Makefile	Tue Jul 23 21:28:49 2002
@@ -130,23 +130,3 @@
 	rm -f *.o
 	
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
--- ./Makefile.org	Tue Jul 23 13:07:49 2002
+++ ./Makefile	Tue Jul 23 21:28:00 2002
@@ -1,15 +1,16 @@
 # Makefile	Sun Jul 23 11:50:12 2002
 
-INSTALL_DIR = /opt/MathPlanner3
+PREFIX		= /usr
+SBINDIR		= $(PREFIX)/sbin
+INSTALL_DIR	= /opt/MathPlanner3
 
-all:
-	@echo "Type 'make install' to build and install MathPlanner"
+all:	build install
+
+build:
+	make -C src all
 
 install:
-	mkdir $(INSTALL_DIR)
-	cp -R * $(INSTALL_DIR)
-	make -C $(INSTALL_DIR)/src all
-	install -s -m 755 $(INSTALL_DIR)/src/MathPlanner3 /usr/bin/MathPlanner3
-	make -C $(INSTALL_DIR)/src clean
+	install -d $(DESTDIR)$(SBINDIR)
+	install -s -m 755 src/MathPlanner3 $(DESTDIR)$(SBINDIR)
 	@echo " "
 	@echo "Type MathPlanner3 to start MathPlanner"
